angr/state_plugins/symbolic_memory.py

```diff
@@ -312,9 +312,12 @@ def make_symbolic(self, name, addr, length=None):
         r = self.load(addr, length)
 
         v = self.get_unconstrained_bytes(name, r.size())
-        self.store(addr, v)
-        self.state.add_constraints(r == v)
-        l.debug("... eq constraints: %s", r == v)
+        self.store(addr, v, add_constraints=add_constraints)
```

Comment:
Not sure if passing `add_constraints=add_constraints` is strictly necessary, I am new to angr :sweat_smile: